atef/util.py

```diff
@@ -0,0 +1,8 @@
+import ophyd
+
+
+def ophyd_cleanup():
+    """Clean up ophyd - avoid teardown errors by stopping callbacks."""
+    dispatcher = ophyd.cl.get_dispatcher()
+    if dispatcher is not None:
+        dispatcher.stop()
```

Comment:
Neat trick

Reply:
Segfaults otherwise for short-lived sessions with trying-to-connect PVs 😬 
I wonder if ophyd doesn't have an atexit hook for stopping the dispatcher - I could have sworn it did.

Reply:
https://github.com/bluesky/ophyd/blob/7937cd46e92e9ec2f7eec321dd5b1b1b71115e20/ophyd/_pyepics_shim.py#L165